test/test_private.py

```diff
@@ -50,7 +50,7 @@ def test_safe_dump(self):
             yaml.safe_dump(YAMLTest())
 
 
-class TestStrconv(unittest.TestCase):
+class TestStrconv:
```

Comment:
Huh, interestingly this name was wrong before -- it should be `Timeconv`, not `Strconv`. However, I think we should probably just move these to the top level too. We could move them into `test_timeconv.py` and `test_yaml.py` -- "flat is better than nested" (Zen of Python). What do you think?

Reply:
I agree, and since YAML and Timeconv don't really have much to do with each other, there isn't a strong reason for them to live in the same test file. This change results in smaller files, too, which I like. I'll do a separate commit for this.